client/services/FeedbackGeneratorService.js

```diff
@@ -52,6 +53,22 @@ tie.factory('FeedbackGeneratorService', [
               ].join(''), false);
             }
           }
+          // TODO(eyurko): Potentially use regression to check runtime complexity, rather than just guessing.
```

Comment:
Why do you need this TODO here? It seems like it's being handled in the preprocessor already, and that this isn't quite the right place for it. (Unless I'm misunderstanding, since I'm not sure what precisely you mean by "use regression to check runtime complexity".)